File: pset6/mario/mario.py
from cs50 import get_int
# test number to make while
n = 1000
while 1 > n or 8 < n:
    n = get_int("Height:")
for i in range(n):
    for m in range(n-(i+1)):
        print(" ", end="")
    for t in range(i+1):
        print("#", end="")
    print("  ", end="")
    for q in range(i+1):
        print("#", end="")
    print()

# Yousof.A.Asadi(Y.A.A)11/24/2021

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n; //number
    int b; //line
    int c; //space
    int e; //#
    do
    {
        n = get_int("Height:  ");
    }
    while (n < 1 || n > 8);

    for (b = 0; b < n; b++)  //line
    {
        for (c = n - (1 + b); c > 0; c--)  //space
        {
            printf(" ");
        }
        for (e = 0; e < b + 1; e++)  //#
        {
            printf("#");
        }
        printf("\n");
    }
}

